litgpt/deploy/serve.py

```diff
@@ -106,6 +121,42 @@ def encode_response(self, output: torch.Tensor) -> Dict[str, Any]:
         return {"output": decoded_output}
 
 
+class StreamLitAPI(BaseLitAPI):
```

Comment:
Why did you prefer to add a new class than adding a flag to control whether the existing  class should stream?

Reply:
This was because of how `yield/return` works in Python (as far as In understand). Unfortunately, once you have a `yield` in a function or method (here the `predict` method and `encode_response` method), it will always return a generator, even if you use `return`. I thought this would be unnecessary complicated to handle for those who don't want to use streaming.